### PR TITLE
Add PR & Issue templates

### PR DESCRIPTION
**Pull request by pyr0gan**
- Added `.github/PULL_REQUEST_TEMPLATE.md`
- Added `.github/PULL_REQUEST_TEMPLATE/contribution_pull_request_template.md`
- Added `.github/PULL_REQUEST_TEMPLATE/guide_pull_request_template.md`
- Added `.github/ISSUE_TEMPLATE/config.yml`
- Added `.github/ISSUE_TEMPLATE/bug.yml`
- Added `.github/ISSUE_TEMPLATE/feature.yml`
- Updated `README.md`
- Small changes to `CONTRIBUTING.md`

### CODE REVIEW

**Comment by Necmttn** [comment]
@pyr0gan for issue templates we can have form you can check the example below
https://github.com/paircodedev/sandbox/issues/new/choose
image
the source code for it: https://github.com/paircodedev/sandbox/tree/main/.github/ISSUE_TEMPLATE

A similar multifile template might also be available for Pull Requests just need to test it.
🙌

File: .github/PULL_REQUEST_TEMPLATE/guide_pull_request_template.md.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Guide Outline
+
+> Provide a one or two paragraph outline of your guide. You should mention the overall topic, and what you expect learners to acheive when completing your guide.
+
+## Link to Proposal
+
+> Include a link to the related proposal on Kafé (e.g. <https://dev.builderdao.io/vote/my-proposal-slug>)
```
Comment (Necmttn) [inline .github/PULL_REQUEST_TEMPLATE/guide_pull_request_template.md.md:7]: link to the proposal is auto-generated by the GitHub action.  for example: 
https://github.com/TheBuilderDAO/kafe/pull/158#issuecomment-1099556450
would appreciate some styling feedback on this one actually @pyr0gan @dgamboa

Comment (pyr0gan) [reply .github/PULL_REQUEST_TEMPLATE/guide_pull_request_template.md.md:7]: I'm not sure what you mean by styling feedback here, @Necmttn 🤔

Comment (Necmttn) [reply .github/PULL_REQUEST_TEMPLATE/guide_pull_request_template.md.md:7]: image
as the wording and utilization of emojis etc.

**Pull request merged**